code/mob/living/carbon/human/procs/get_desc.dm

```diff
@@ -90,15 +91,17 @@
 		else
 			. += "<br><span class='[src.r_hand.blood_DNA ? "alert" : "notice"]'>[src.name] has [bicon(src.r_hand)] [src.r_hand.blood_DNA ? "a bloody [src.r_hand.name]" : "\an [src.r_hand.name]"] in [t_his] right hand.</span>"
 
-	if (src.belt)
+	if (src.belt && !(src.wear_suit?.hides_from_examine & C_BELT))
```

Comment:
We don't want belts to be hideable by clothing.

Reply:
alrighty